pennylane/ops/op_math/symbolicop.py

```diff
@@ -47,6 +47,10 @@ class SymbolicOp(Operator):
 
     _name = "Symbolic"
 
+    @classmethod
+    def _primitive_bind_call(cls, *args, **kwargs):
```

Comment:
This could do with a comment on why it seemingly is defined to behave trivially.